modelforge/potential/processing.py

```diff
@@ -334,3 +421,92 @@ def forward(self, data: Dict[str, torch.Tensor]) -> torch.Tensor:
 
         data["ase_tensor"] = ase_tensor
         return data
+
+
+class LongRangeElectrostaticEnergy(torch.nn.Module):
+    def __init__(self, strategy: str, cutoff: unit.Quantity):
+        """
+        Computes the long-range electrostatic energy for a molecular system
+        based on predicted partial charges and pairwise distances between atoms.
+
+        The implementation follows the methodology described in the PhysNet
+        paper, using a cutoff function to handle long-range interactions.
```

Comment:
Probably good to note that that this uses a cosine cutoff function, not the original function in physnet.

Reply:
that has been resolved